service/src/main/java/bio/terra/workspace/service/iam/SamService.java

```diff
@@ -207,10 +199,10 @@ public UserStatusInfo getUserStatusInfo(AuthenticatedUserRequest userRequest)
   public String getOrCreateUserManagedIdentityForUser(
       String userEmail, String subscriptionId, String tenantId, String managedResourceGroupId)
       throws InterruptedException {
-    AzureApi azureApi = samAzureApi(getWsmServiceAccountToken());
 
-    GetOrCreatePetManagedIdentityRequest request =
-        new GetOrCreatePetManagedIdentityRequest()
+    AzureApi azureApi = samAzureApi(getWsmServiceAccountToken());
+    GetOrCreateManagedIdentityRequest request =
```

Comment:
So is this actually no longer a pet identity?

Reply:
Pet identities (a Terra concept) are a subset of User Assigned Managed Identities (an Azure concept). This type encompasses the Azure concept, not the Terra one. 

The new Sam client renamed this type since now Sam knows about more than just pet identities, e.g. 'action identities.' Renaming the type allows it to use the same logic for things that are identical on the Azure side of things. 

This method is still retrieving the user's pet identity and calling the Sam pet endpoint.